changes/3652-dolfinus.md

```diff
@@ -0,0 +1 @@
+Port number is no longer being ignored by `HttpUrl.build()`
```

Comment:
sorry for the delay, this description is a bit unclear - when reading it I'm not sure whether this is the "no longer being ignored" is the bug or the fix state.

Maybe you could tweak it to be clear about the new behaviour.